wordpress-mobile/WordPress-iOS
# Unified About: App Logos

Fixes #17405. This PR adds a cell containing logos of the various Automattic apps. This is rendered with SpriteKit and the apps randomly fall into place using physics bodies. It should also handle dark mode correctly. Originally the logos responded to the motion of the user's device but I can't currently find a way to make this work nicely when the app can rotate to any orientation, so I've taken the code out for now.

| Light Mode | Dark Mode |
|---|---|
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-11-04 at 16 10 59](https://user-images.githubusercontent.com/4780/140376886-87500028-13eb-4e78-980e-9bf7e35850a4.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-11-04 at 16 12 49](https://user-images.githubusercontent.com/4780/140376905-61cd06ad-f80f-41a8-a281-018bed8c10fa.png) |

On wider devices like Max phones or iPad, I've restricted the maximum width of the SpriteKit area, otherwise things were looking very sparse with a few balls spread out in a line over a wide distance:

| Before | After |
|---|---|
| ![image (4)](https://user-images.githubusercontent.com/4780/140377254-f0608916-5e90-4701-a818-0f4d624a0301.png) | ![Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2021-11-04 at 13 29 24](https://user-images.githubusercontent.com/4780/140377199-21041435-4bec-4d29-8324-95e9d44eb03e.png) |

**To test**

* Build and run
* Navigate to about, check the logos look good. Try tapping on some too, they bounce a little.
* Test on a range of device sizes, rotate the screen, and check light and dark mode.

## Regression Notes

1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
